.github/CODEOWNERS

```diff
@@ -71,6 +71,9 @@ gradlew.bat @rt4914
 # Linter configuration.
 buf.yaml @anandwana001
 
+# Maven artifacts pinning.
+maven_install.json @BenHenning
```

Comment:
Per my feedback in your document today, could we maybe put this in third_party, or do we not have a choice?

Reply:
I tried moving it to third_party package by following the [documentation](https://github.com/bazelbuild/rules_jvm_external#custom-location-for-maven_installjson) but it is throwing error - `Could not find a BUILD file`

![Screenshot from 2021-06-22 11-52-32](https://user-images.githubusercontent.com/54636525/122874694-e6535f00-d350-11eb-8ba3-d9f80c978c7d.png)

I also tried making the same location as mentioned in the documentation (by adding name attribute and making a package in the third-party directory along with an empty BUILD file) but it threw the same error.
![Screenshot from 2021-06-22 12-01-13](https://user-images.githubusercontent.com/54636525/122875250-a2148e80-d351-11eb-8ddd-b894551ecc1d.png)

Reply:
Just left another comment--I think you might have made a mistake in the WORKSPACE file.

Reply:
Even after changing this, I am getting the same error (I already tried this earlier).

Reply:
Thanks, it got fixed. I also had to shift it to third_party manually.